internal/cmd/util/validation.go

```diff
@@ -0,0 +1,44 @@
+package util
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var posArgRegex = regexp.MustCompile(`\[.*?]|\(.*?\)|--.*?<.*?>|<(.*?)>`) // matches all positional args in group 1
```

Comment:
I find the use of a regex for such feature a bit fragile, why didn't you choose to have a list of args and flags that could be both rendered to a usage string and used to validate the missing args ?

Reply:
There is also https://github.com/docopt/docopt.go that might be used to get the info we want.

Reply:
Rendering args and flags to a usage string could be complicated in some cases, for example [here](https://github.com/hetznercloud/cli/pull/711/files#diff-37acc61009ac872d74e2bbfe664618f99e6ba1e9497c01dd65f1fd3f4a5553b7R19) and [here](https://github.com/hetznercloud/cli/pull/711/files#diff-09ce77c4101f651b597cdb7b6f14a4252ad407a7b0999abe9bf6cc8419ffdd5aR19). Parsing seems like the best bet. Maybe we can add a comment to explain the regex.